Decred Wallet/DcrlibwalletTypes/Wallet.swift

```diff
@@ -14,19 +14,33 @@ class Wallet: NSObject {
     private(set) var name: String
     private(set) var balance: String
     private(set) var accounts = [DcrlibwalletAccount]()
+    private(set) var visibleAccounts = [DcrlibwalletAccount]()
     private(set) var isSeedBackedUp: Bool = false
     private(set) var displayAccounts: Bool = false
+    private(set) var isRestored: Bool = false
+    private(set) var hasDiscoveredAccounts: Bool = false
```

Comment:
Should assign these in the constructor function. Don't think these will be necessary though, so can be removed.